static/js/message_list_view.js

```diff
@@ -257,11 +257,16 @@ export class MessageListView {
         if (last_edit_timestamp !== undefined) {
             const last_edit_time = new Date(last_edit_timestamp * 1000);
             const today = new Date();
-            return (
-                timerender.render_date(last_edit_time, undefined, today)[0].textContent +
-                " at " +
-                timerender.stringify_time(last_edit_time)
-            );
+
+            let last_edit_date = timerender.render_date(last_edit_time, undefined, today)[0]
+                .textContent;
+
+            if (last_edit_date === "Today") {
+                last_edit_date = "today";
+            } else if (last_edit_date === "Yesterday") {
+                last_edit_date = "yesterday";
+            }
+            return last_edit_date + " at " + timerender.stringify_time(last_edit_time);
```

Comment:
![image](https://user-images.githubusercontent.com/82862779/199787157-d047511f-9bc7-431a-8795-85d7087ed6bd.png)

@alya while showing actual dates, the requested format of Last {moved/edited} {date} sounds a bit odd, I think we need to add "on" before the date when we show the actual date.

Reply:
Hm, I'm not sure -- I think it reads OK as-is, so I wonder if it's best to keep it simple (both in terms of the logic, and even more so for translation purposes). @timabbott what do you think?

Reply:
I think it's OK as-is.

As a sidenote, using `+` to combine string fragments is not a correct way to write translatable strings; see 
https://zulip.readthedocs.io/en/latest/translating/internationalization.html for background. But generally one needs to have an entire sentence be a single translateable string.

Reply:
Here, instead of '+' , I constructed a single string using '`' template literals.